examples/with-aws-amplify-typescript/next-env.d.ts

```diff
@@ -3,3 +3,8 @@
 
 // NOTE: This file should not be edited
 // see https://nextjs.org/docs/basic-features/typescript for more information.
+
+declare module '*.css' {
```

Comment:
I don't think this file should be edited - is there another way to do this?

Reply:
You can do this with `const styles = require("../styles/Home.module.css");` instead of `import "../styles/Home.module.css"` in index.tsx and [id].tsx files. The thing is it will not compile with just simple `import "../styles/Home.module.css"`. Should I change it?